win32/tests/mocked/inc/windows.h

```diff
@@ -0,0 +1,78 @@
+// Copyright(C) Microsoft Corporation.All rights reserved.
```

Comment:
Is this file being used? We should remove it now if possible, or make changes later to remove it